pipelines/konflux-e2e-tests.yaml

```diff
@@ -0,0 +1,279 @@
+              - --timeout
+              - "10m0s"
+              - "$(params.bundleImage)"
+    - name: e2e-test
+      description: Placeholder task that prints the Snapshot and outputs standard TEST_OUTPUT
+      runAfter:
+        - deploy-operator
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+      taskSpec:
+        params:
+          - name: SNAPSHOT
+        volumes:
+          - name: credentials
+            emptyDir: { }
+        results:
+          - name: TEST_OUTPUT
+            description: Test output
+        steps:
+          - name: get-kubeconfig
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/build-definitions.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml
+            params:
+              - name: eaasSpaceSecretRef
+                value: $(tasks.provision-eaas-space.results.secretRef)
+              - name: clusterName
+                value: "$(tasks.provision-cluster.results.clusterName)"
+              - name: credentials
+                value: credentials
+          - name: e2e
+            image: quay.io/openshift-pipeline/ci:latest
+            env:
+              - name: SNAPSHOT
+                value: $(params.SNAPSHOT)
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
+            script: |
+              #!/usr/bin/env bash
+              set -ex -u -o pipefail
+              
+              NAMESPACE="default" # using default since openshift-builds NS is not available
+              DEPLOYMENTS=("openshift-builds-operator" "openshift-pipelines-operator" "shipwright-build-controller" "shipwright-build-webhook" "tekton-operator-webhook")
+              BUILDS=("buildah-golang-buildrun")
+
+              oc get all -n $NAMESPACE
+              # sleep for 5 minutes, give deployments time to get ready
+              sleep 300
+              oc get all -n $NAMESPACE
+              for DEPLOYMENT in "${DEPLOYMENTS[@]}"; do
+                  oc wait deployment "$DEPLOYMENT" --namespace="$NAMESPACE" --for="condition=available" --timeout="180s"
+              done
+
+              echo "All specified deployments are running and ready."
+
+              # Check running builds
+              oc apply -f - <<EOF
+              apiVersion: shipwright.io/v1beta1
+              kind: Build
+              metadata:
+                name: buildah-golang-build
+              spec:
+                source:
+                  type: Git
+                  git:
+                    url: https://github.com/shipwright-io/sample-go
```

Comment:
There is samples repository in our org also. 
https://github.com/redhat-openshift-builds/samples
Can this be used?

Reply:
Yes, let me change and rerun the tests.